src/main/java/core/basesyntax/service/FruitShopService.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.FruitShopInventory;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationStrategy;
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+
+public class FruitShopService {
+
+    private final ReportWriter reportWriter = new ReportWriter();
+    private final FruitShopInventory inventory = new FruitShopInventory();
+    private final DataConverter dataConverter = new DataConverter();
+    private final Map<FruitTransaction.OperationType, OperationStrategy> operationsMap;
```

Comment:
not fixed. Move these dependencies to the Main class. See example in the readme file